CJBCheatsMenu/i18n/pt.json

```diff
@@ -9,7 +9,7 @@
     "tabs.player-and-tools": "Jogador e Ferramentas",
     "tabs.farm-and-fishing": "Cultivo e Pesca",
     "tabs.skills": "Habilidades",
-    "tabs.weather": "Tempo",
```

Comment:
"Tempo" in portuguese means both time and weather, so this should make it more clear